flexget/plugins/filter/series.py

```diff
@@ -313,7 +373,11 @@ class Episode(Base):
 
     identified_by = Column(String)
     series_id = Column(Integer, ForeignKey('series.id'), nullable=False)
-    releases = relation('Release', backref='episode', cascade='all, delete, delete-orphan')
+    releases = relation('EpisodeRelease', backref='episode', cascade='all, delete, delete-orphan')
+
+    @property
+    def is_season(self):
+        return False
```

Comment:
We're using it in a bunch of places actually, like https://github.com/Flexget/Flexget/blob/5b2988d31cdb980b546b490120bd532d81791663/flexget/plugins/filter/series.py#L1486

What's the benefit of setting a class variable as opposed to a property method?